app/org/sagebionetworks/bridge/play/controllers/AuthenticationController.java

```diff
@@ -102,7 +87,40 @@ public Result resetPassword() throws Exception {
         authenticationService.resetPassword(passwordReset);
         return okResult("Password has been changed.");
     }
-    
+
+    private Result signInWithRetry(final int retryCounter) throws Exception {
```

Comment:
Minor: Add comment that retryCounter is the number of retries, excluding the initial try. For example, if retryCounter=3, there will be 4 tries total.